mirage/factories/user.ts

```diff
@@ -12,7 +20,7 @@ export default Factory.extend({
         return faker.name.firstName();
     },
     middleNames() {
-        return `${faker.name.firstName()} ${faker.name.firstName()}`;
+        return [`${faker.name.firstName()} ${faker.name.firstName()}`];
```

Comment:
The API returns a string. Why is this an array?

Reply:
Maybe our Ember user model is wrong then?